CMakeLists.txt

```diff
@@ -130,6 +130,15 @@ if(LINALG_ENABLE_KOKKOS)
   find_package(KokkosKernels REQUIRED)
 endif()
 
+find_package(TBB)
```

Comment:
More recent versions of GCC (for instance) shouldn't require TBB for `std::execution::par` to work.  Furthermore, nvc++ comes with its own, non-TBB implementation of `std::execution::par`.  Would you consider gating this on compiler version, instead of requiring a third-party library?  That could even be done in the code -- the code would just need to test the appropriate compiler version to ensure that the C++ algorithms are available (see https://en.cppreference.com/w/cpp/compiler_support/17 and search for "Parallel algorithms and execution policies").